src/views/partials/admin/footer.ejs

```diff
@@ -1,2 +1,3 @@
 <!-- views/partials/admin/footer.ejs -->
-<p>Dit is het einde van de pagina</p>
+<link rel="stylesheet" href="/css/shared.css" />
```

Comment:
Waarvoor is dit?

Reply:
omdat ik vergeten was dat dit al in de head patrial stond :)